chain/default_store.go

```diff
@@ -373,6 +373,11 @@ func (store *DefaultStore) Head() types.TipSet {
 	return store.head
 }
 
+// BlockHeight returns the height of the head
```

Comment:
Please clarify whether "height" counts non-null blocks/tipsets, or rounds (i.e. including "null blocks"). I assume the former, based on conversations with others, with "round number" or "ticket number" being the latter. But I fear it is in fact the latter.